### ISSUE TITLE
Telemetry Implementation

### ISSUE THREAD

**Issue by coolbung** [shiksha-platform/frontend-modulefederation#36]
Let's implement posthog (https://posthog.com/) for telemetry tracking in the frontend. 

The SDK documentation https://posthog.com/docs/integrate/client/js#autocapture seems to indicate that just adding the SDK will automatically start tracking all pageviews and click events. 

@Shruti3004 @arajput can you verify if this will work in react, and if additional effort is needed to make it work with module federation ?

@rahul101001000 @SaketSinha

**Comment by rahul101001000** [comment]
Yes let's do it. In the future, once Sunbird telemetry is in better form we could move to that. 

Have an abstraction layer like Google Tag Manager or Segment open source equivalent so that switch to any telemetry provider is easy. Let the deployer have that choice.
